asf_heat_pump_suitability/pipeline/prepare_features/land_extent.py

```diff
@@ -0,0 +1,163 @@
+import pandas as pd
+import geopandas as gpd
+import regex as re
+from typing import List
+from asf_heat_pump_suitability import config
```

Comment:
This should work because [asf_heat_pump_suitability/asf_heat_pump_suitability/__init__.py](https://github.com/nestauk/asf_heat_pump_suitability/blob/dev/asf_heat_pump_suitability/__init__.py) returns config/base.yaml as a which can then be subscripted. It's part of the DS cookiecutter I think?

Reply:
Apologies, I must have broken this somehow. I restarted my kernel and it works fine!